### ISSUE TITLE
⚠️Important⚠️ - Sign up registration code is not written properly/broken

### ISSUE THREAD

**Issue by haydenkong** [mckaywrigley/chatbot-ui#1617]
I just tired deploying ChatbotUI to Vercel and I have followed the instructions. 

However, when I tried to signup, it would reload the page (with no prompt, even though i enabled it through the code) and it would send me the confirmation email. It all seemed normal, until when I clicked on the confirmation url. 

It would take my back to the root of the site and then I pressed "Start Chatting", it _**would go straight to the dashboard UI without the onboarding**._ The dashboard UI won't function, you can't type anything and can't view your profile, and I'm pretty sure that's because I haven't done the onboarding and even though I confirmed my email, I am still not a "user" in the "profile" table. 

This needs to be fixed ASAP and I will try to look into it more.
@mckaywrigley 
Thanks :)

**Comment by Virtual-Educator** [comment]
I'm experiencing the same issue.

**Comment by haydenkong** [comment]
Yeah this is a huge issue @mckaywrigley can you please look into it more.
Thankyou!

**Comment by Virtual-Educator** [comment]
Do you have groq_api_key in a column of your Profile table?

**Comment by haydenkong** [comment]
Yes, i do.

**Comment by spammenotinoz** [comment]
I had this problem, my workaround was to remove the api setup page.

**Comment by hannesrudolph** [comment]
Oh yeah I recall this a while back I just manually created a user in the database.

**Comment by saqks-ahm** [comment]
Same here

**Comment by haydenkong** [comment]
@spammenotinoz if you remove the api setup page, it still doesn't work 


> I had this problem, my workaround was to remove the api setup page.

**Comment by PixelVerseIT** [comment]
> Oh yeah I recall this a while back I just manually created a user in the database.

Yeah i guess that works but it's just easier and more simpler to signup on the website, plus other people could  use it as well.

**Comment by haydenkong** [comment]
I noticed the official ChatBotUI signup works perfectly fine? 

@mckaywrigley could you please share with us if you have made any changes to the official ChatBotUI so that the signup works? Thanks.
